docs/Using Fleet/Log-destinations.md

```diff
@@ -16,8 +16,56 @@ This document provides a list of the supported log destinations in Fleet.
 
 Log destinations can be used in Fleet to log:
 - Osquery [status logs](https://osquery.readthedocs.io/en/stable/deployment/logging/#status-logs).
+
+  
+Example:
+```json
+{
+  "hostIdentifier":"85E0BF4B-E1CD-5AD1-88D1-0DC9220FF6D5",
+  "calendarTime":"Tue Oct  3 20:11:13 2023 UTC",
+  "unixTime":"1696363873",
+  "severity":"0",
+  "filename":"auto_constructed_tables.cpp",
+  "line":"241",
+  "message":"ATC table: tcc_system_entries Registered",
+  "version":"5.9.1"
+}
+```
+
 - Osquery [schedule query result logs](https://osquery.readthedocs.io/en/stable/deployment/logging/#results-logs).
-- Fleet audit logs. 
+
+  
+Example:
+```json
+{
+  "snapshot": [
+    {
+      "model": "USB Keyboard",
+      "vendor": "SIGMACHIP"
+    },
+    {
+      "model": "YubiKey OTP+FIDO+CCID",
+      "vendor": "Yubico"
+    }
+  ],
+  "action": "snapshot",
+  "name": "pack/Global/Get USB device type - 107 - Jul 17 17:47:18.000",
```

Comment:
What's the general format? `(pack/schedule)/(Global/?)/(query name) - (xyz?) - (date run)`?